### PR TITLE
The next web recreation using HTML-CSS

### PR DESCRIPTION
**Pull request by notrexxx**
This is a copy of thenextweb.com using Grid, Flexbox, Media Queries and gradients

### CODE REVIEW

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: require changes :recycle: 

Hi, Great job guys :clap: :clap: :clap: but there are some small changes :toolbox: :hammer: :wrench: that you need to take care of, in order for this project to get approved :checkered_flag: 

I loved :heart: how you did handle different CSS file, for each viewport, great job :tophat: 

Please check the comments below:
:point_down: :point_down: :point_down: :point_down: :point_down:

## Required Changes :red_circle::

- [x] Write a proper description for your project on the readme file, give it a proper name, and also use good screenshots of the project:
image

- [x] Fix the screenshot image link in your readme
image

- [x] Fix your profile links on the readme.

- [x] Fix the header on desktop view:

|*Clone*|*Original*|
|---|----|
| image | image |

- It contains main navigation row
- It contains sub-navigation row
- It contains a search icon at the rightmost side of sub-navigation row
- There should be no logo on the left side corner please follow the original design.

- [x] Fix the header on tablet view:

|*Clone*|*Original*|
|:----:|:-------:|
| image | image |

- [ ] Fix the header on Mobile view they should match correctly:

|*Clone*|*Original*|
|:-----:|:------:|
| image | image |

- [x] Fix the hero section align correctly the title inside each image like it is on the original page:

|*Clone*|*Original*|
|:----:|:----:|
| image | image |

- also you should have a tnw logo on the bigger image on the left.

- [x] align the latest news correctly with the articles and give it a correct size:

|*Clone*|*Original*|
|:----:|:----:|
| image | image |

- [x] Fix the latest funding section on the desktop view, align correctly the headlines and also you are messing another article there are supposed to be 4 articles:

|*Clone*|*Original*|
|:-----:|:------:|
| image | image |

- 4 articles on desktop view
- 3 articles on tablet view
- and 1 article per row on a mobile view

- [x] on the section below the latest funding section you should have three articles per row on desktop view please fix it(on mobile and tablet view you are good):

|*Clone*|*Original*|
|:----:|:-----:|
| image | image |

- [x] Fix the headlines on the latest deals section on desktop and mobile views:

|*Clone*|*Original*|
|:-----:|:------:|
| image | image |

- [ ] The articles below the latest funding section are going out of the screen please position them correctly and also resize them correctly:
image

## Optional:

- [ ] Change the icons into white icons:
image

- [ ] add the off percent on the latest deals section articles:
image

### Please make all the required changes before you submit another code review.
## Happy Coding :tophat: 
_If you have any questions or need help don't hesitate to contact me on slack, I would love to help_
## _Zakariae El Mejdki_

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team     
#### Great Job so far, everything works fine, but there are a few more things required. 
### General

- [x] Let's fix this minor issue, you are to have just three items not four
 
##### Yours

##### Original



### Make changes and submit another code review request.
*Happy coding* 
(Any questions Feel free to contact me on slack @kobiyoyo)

**Comment by notrexxx** [comment]
> ## Hi Team 
> #### Great Job so far, everything works fine, but there are a few more things required.
> ### General
> * [x]  Let's fix this minor issue, you are to have just three items not four
> 
> ##### Yours
> 
> 
> ##### Original
> 
> 
> ### Make changes and submit another code review request.
> _Happy coding_
> (Any questions Feel free to contact me on slack @kobiyoyo)

Hello!

Thank you so much for the input.

I have hidden the investment tab on the mobile viewport and committed it!.

Regards.

**Review by mahmoud717 — CHANGES_REQUESTED** [review]
### Changes required ♻️ 

**Hello @notrexxx  & @EroAuditore**

Great job with the project so far it looks amazing, but there are still some changes to be made:
 
---
### Requested 🔴 

- [x] make sure that you get the right color for the icons in the navbar.
- [x] in the original page there is a margin on both sides of the page but in yours, there isn't. you can create the same effect by giving the main divs a max-width and setting the margin to auto. 


---

please make these changes and request another code review.

if you have any questions don't hesitate to contact me on slack **MahmoudMohammad**

File: styles/desktop.css [hunk 1]
```diff
@@ -0,0 +1,817 @@
...
+  top: 0;
+  left: 0;
+  z-index: 10;
+}
+
+.story-investmentCard-name {
+  background: #f42;
+  color: #fff;
+  padding: 10px 6px 10.6666666667px 84px;
+  margin: 0 !important;
```
Comment (umairarshadbutt) [inline styles/desktop.css]: Remove `!important` from complete CSS files

File: styles/mobile.css [hunk 2]
```diff
@@ -0,0 +1,746 @@
...
+  left: 0;
+  z-index: 10;
+}
+
+.story-investmentCard-name {
+  background: #f42;
+  color: #fff;
+  padding: 10px 6px 10.6666666667px 84px;
+  margin: 0 !important;
+  font-size: 18px !important;
```
Comment (umairarshadbutt) [inline styles/mobile.css]: Remove `!important` from complete CSS files

File: styles/tablet.css [hunk 3]
```diff
@@ -0,0 +1,773 @@
...
+  left: 0;
+  z-index: 10;
+}
+
+.story-investmentCard-name {
+  background: #f42;
+  color: #fff;
+  padding: 10px 6px 10.6666666667px 84px;
+  margin: 0 !important;
+  font-size: 18px !important;
```
Comment (umairarshadbutt) [inline styles/tablet.css]: Remove `!important` from complete CSS files

File: index.html [hunk 4]
```diff
@@ -0,0 +1,1143 @@
...
+          <a href="#">
+            <i class="fas fa-tag"><span class="font-fs inv-space"> Information Technology, +2</span></i>
+          </a>
+          <a href="#">
+            <i class="fas fa-briefcase"> <span class="font-fs inv-space"> Battery Ventures, +6</span></i>
+          </a>
+        </div>
+
+        <div class="card-story__footer font-fs">
+          <a href="">
```
Comment (umairarshadbutt) [inline index.html]: Add `#` in all empty `href` attributes

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @notrexxx and @EroAuditore 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [x] Remove `!important `from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)
- [x] Live link is not working in README file
- [x] Add `#` in all empty `href` attributes

### Navbar
- [x] Please fix the responsive issues

|[Source Page](https://thenextweb.com/)|[Clone Page](https://rawcdn.githack.com/notrexxx/TNW-RESPONSIVE-HTML-CSS/Responsive/wip/index.html)|
|:---:|:---:|
|source-page|clone|


##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Comment by notrexxx** [comment]
> # Status: REQUIRED CHANGES ⚠️ 🚧
> Hi @notrexxx and @EroAuditore
> Good job with the changes! 🎉 We are almost there.
> We, however, do have a few changes to carry out before we proceed (see review comment).
> 
> For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
> 
> * [x]  Remove `!important `from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)
> * [x]  Live link is not working in README file
> * [x]  Add `#` in all empty `href` attributes
> 
> ### Navbar
> * [x]  Please fix the responsive issues
> 
> [Source Page](https://thenextweb.com/)	[Clone Page](https://rawcdn.githack.com/notrexxx/TNW-RESPONSIVE-HTML-CSS/Responsive/wip/index.html)
> source-page	clone
> * **When you have made the specified changes, please submit another code review. Thank you!**
> 
> **If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
> HAPPY CODING 💻
> **[Umair Arshad](https://umairarshad.me/)**
> [Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

LInk is live

**Review by ElijahTobs — APPROVED** [review]
# Status: Approved! :tada: 

## Congratulations Team :medal_sports:  

Good job implementing the changes requested by the previous TSEs.

## You can now merge and submit.

## Happy Coding :computer: 

_Feel free to reach out to me on slack_ **`@Elijah Ayandokun`**
  **Elijah O. Ayandokun**
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

**Pull request merged**